common/changes/@itwin/appui-react/mast-lodash-import-fix_2025-01-17-12-23.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/appui-react",
+      "comment": "fix lodash imports from named to default for compatability with esm",
```

Same for other change files.
```suggestion
      "comment": "Fixed `module 'lodash' does not provide an export named...` error when running in Node.js.",
```

---

changed